packages/manager/apps/dedicated/client/app/components/ip/agoraOrder/ip-ip-agoraOrder.controller.js

```diff
@@ -388,6 +397,52 @@ export default class AgoraIpOrderCtrl {
     return this.$state.go('^');
   }
 
+  getOfferDetails = (offerDetails, ipOffersByRegion, countryList) => {
+    return offerDetails
+      .filter(({ productRegion }) => ipOffersByRegion.includes(productRegion))
+      .map((ipOffer) => {
+        set(
+          ipOffer,
+          'productDisplayName',
+          `${ipOffer.productShortName} - ${ipOffer.price.text}`,
+        );
+        set(
+          ipOffer,
+          'countries',
+          ipOffer.countries.filter(
+            ({ code }) => countryList.indexOf(code.toLowerCase()) > -1,
+          ),
+        );
+        return ipOffer;
+      });
+  };
+
+  filterOffer = (ipOfferDetails, nameKey) => {
```

Comment:
Any particular reason to use function expression here?

Reply:
There is a way to avoid getting the lint message. In the function we have VPS_MAX_QUANTITY, just declare it in the init this.VPS_MAX_QUANTITY = VPS_MAX_QUANTITY;
Then you have a this in the function.